platform/src/lib/store/airQloudsApi.js

```diff
@@ -0,0 +1,36 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { AIRQLOUDS } from '@/core/urls/airqlouds';
+import { HYDRATE } from 'next-redux-wrapper';
+import { NEXT_PUBLIC_AUTHORISATION } from '../envConstants';
```

Comment:
the variable `NEXT_PUBLIC_AUTHORISATION` has a Next.js prefix that allows this value to be seen by anyone through the browser.

Reply:
For now we are using this approach since we don't have a login system and the token wont be accessible by the browser except if I add the nextjs prefix

Reply:
Hi @Codebmk and @123MwanjeMike , thank you so so much for the amazing works so far. Can't we use the existing login backend system (endpoints) for the Legacy platform.

As part of our original approach for the Platform redesign, we are not creating everything from scratch, we need to take advantage of existing endpoints as much as possible. We are not throwing away the backend of the Legacy system. In fact, we are likely to utilise close to 100% of the Legacy system's backend!

Otherwise, we shall never move and it will frustrate everyone involved. Also, we need to explore the micro-frontend architecture principles as we proceed on this endeavour.
 
Looking forward to further developments around this. Thanks!

Reply:
@Codebmk / @123MwanjeMike 
In summary, we are not building a "new" system! We are just redesigning the existing system so we should move faster since we have all the infrastructure and almost all endpoints in place.

Reply:
True @Baalmart . The login implementation is a separate task and will be implemented in the next sprint. With the remaining time left in this sprint, it is best to focus on the goal we set out to accomplish--search.

Reply:
Great @Codebmk 
I hope we can also have the corresponding tests written as well.
Thanks